features/speculation-rules.yml.dist

```diff
@@ -4,8 +4,8 @@
 status:
   baseline: false
   support:
-    chrome: "127"
-    chrome_android: "127"
+    chrome: "136"
+    chrome_android: "136"
```

Comment:
I let this one advance, since there's one implementer. Open to reconsideration on this.